### PR TITLE
CIV-926 Respond written resp handler and unit test

### PR DESCRIPTION
**Pull request by karthick-mohanasundaram-HMCTS**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-926

### Related PRs ###

https://github.com/hmcts/civil-general-apps-ccd-definition/pull/145

### Change description ###

This enables applicants/respondents (as required) to upload documentation that has been requested by the judge.

No status change needs to take place on the submission of any documents.

Tasks that need to be displayed for applicant/respondent on Application Case...

Respond to judges directions - if application is in status 'Awaiting Directions Order Docs' (AWAITING_DIRECTIONS_ORDER_DOCS)

Respond to judges written representations - if application is in status 'Awaiting Written Representations' (AWAITING_WRITTEN_REPRESENTATIONS)

Respond to judges additional information - if application is in status 'Awaiting Additional Information' (AWAITING_ADDITIONAL_INFORMATION)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Review by smitatalawadekar-hmcts — APPROVED** [review]

**Review by vasudevganesanhmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/enums/CaseState.java [hunk 1]
```diff
@@ -24,7 +24,7 @@ public enum CaseState {
     AWAITING_DIRECTIONS_ORDER_DOCS("Directions Order Made"),
     ORDER_MADE("Order Made"),
     LISTING_FOR_A_HEARING("Listed for a Hearing"),
-    AWAITING_WRITTEN_REPRESENTATIONS("Written Representations Required"),
+    AWAITING_WRITTEN_REPRESENTATIONS("Awaiting Written Representations"),
```
[the remark this answers is not in the document]
Comment (karthick-mohanasundaram-HMCTS) [reply src/main/java/uk/gov/hmcts/reform/civil/enums/CaseState.java:27]: Hey Smita, Yes, I made the corresponding changes before. Please take a look at the attached image below
image

**Pull request merged**